addons/koji/common/src/main/java/org/commonjava/indy/koji/data/KojiRemovePemMigrationAction.java

```diff
@@ -76,7 +76,7 @@ private boolean doMigrate() throws IndyLifecycleException
         List<RemoteRepository> repos;
         try
         {
-            repos = storeDataManager.query().packageType( PackageTypeConstants.PKG_TYPE_MAVEN ).getAllRemoteRepositories();
+            repos = storeDataManager.query().noPackageType().getAllRemoteRepositories();
```

Comment:
This will introduce the "generic-http" type which is a huge number. If we can confirm all koji remote is maven type, I think leave maven filter can be considered.

Reply:
Yeah, that's what i supposed, but when I test locally yesterday, it returned nothing when specify the package type maven, I think that maybe a bug on the query side, need to figure it out later.

Reply:
After checking the code, I found that it's empty when specifying the packageType, because it queries the record from the cache `CacheHandle<String, Map<StoreType, Set<StoreKey>>> storesByPkg`, which is initialized in startUpAction and executed after the migrateAction. So it's fine to query all remote repos here, for the one time migration.